.github/workflows/hpu-gaudi2.yml

```diff
@@ -39,13 +39,14 @@ jobs:
     # The type of runner that the job will run on
     runs-on: [self-hosted, intel, gaudi2]
     container:
-      image: vault.habana.ai/gaudi-docker/1.15.1/ubuntu22.04/habanalabs/pytorch-installer-2.2.0:latest
+      image: vault.habana.ai/gaudi-docker/1.16.2/ubuntu22.04/habanalabs/pytorch-installer-2.2.2:latest
```

Comment:
@vshekhawat-hlab - this is updated, here is a sample of it running before this PR

https://github.com/microsoft/DeepSpeed/actions/runs/10374148134/job/28721933618#step:4:65